### DISCUSSION TITLE
Conditionals in a Fluid Field Group in CP

### DISCUSSION THREAD

**Discussion by clifavenu** [ExpressionEngine/ExpressionEngine#3546] [Feature Requests]
### Suggest an idea for ExpressionEngine

Hi,
The ability to have conditionals work in a fluid field group would allow more advanced groups instead of multiple simpler ones to accomplish the same task, Grouping and modifying related fields would help intuitive use as well reducing screen clutter for the editors.

Thanks,
clif

**Comment by intoeetive** [comment]
@clifavenu sorry that it has been a while since you posted the issue - but would you be able to clarify what exactly your request is?

It's not complete clear what you suggest, few examples might be useful

**Comment by clifavenu** [comment]
Hi Yuri,

We're looking for conditionals to be available inside a fluid group. For 
example:

1] a fluid group consisting of text input, wysiwyg and file (images).  
If we had conditionals functioning, we could do the following:
2] add a toogle to turn the wysiwyg on / off.

Basically the same process as currently available outside of the fluid 
group.

Not sure if this explains it well enough. Please get back for further 
clarification if needed.

Thanks,
clif

**Reply by intoeetive** [reply]
I am getting impression that you speak about Conditional Fields in CP (but within Fluid) and not conditionals in front-end templates - is that correct?

**Comment by clifavenu** [comment]
Hi Yuri,

Yes, I believe we're on the same page.

Thanks,
clif

**Comment by intoeetive** [comment]
Another request for this: https://github.com/ExpressionEngine/ExpressionEngine/issues/4440
